cluster-scope/overlays/nerc-ocp-prod/services/router-external-apps-ingress-controller.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    metallb.universe.tf/address-pool: public
+    traffic-policy.network.alpha.openshift.io/local-with-fallback: ""
+  labels:
+    app: router
+    ingresscontroller.operator.openshift.io/owning-ingresscontroller: external-apps-ingress-controller
+    router: router-external-apps-ingress-controller
+  name: router-external-apps-ingress-controller
+  namespace: openshift-ingress
+spec:
```

Comment:
We need to add the `domain` option to this configuration; see [the ingress docs](https://docs.openshift.com/container-platform/4.10/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding-route-labels_configuring-ingress-cluster-traffic-ingress-controller).

The domain should be `apps.shift.nerc.mghpcc.org`.